src/custom-element.js

```diff
@@ -1348,6 +1348,25 @@ function createBaseCustomElementClass(win) {
       }
     }
 
+    /**
+     * Get the consent policy to follow.
+     * @return {?string}
+     */
+    getConsentPolicy_() {
+      const policyId = this.getAttribute('data-block-on-consent');
+      if (policyId === null) {
+        // data-block-on-consent attribute not set
+        return null;
+      }
+      if (policyId == '') {
```

Comment:
Note: this will be a breaking change for docs with `data-block-on-consent=default' now. We did document to not set value to the `data-block-on-consent` attribute, however I know there're usage of `data-block-on-consent=default` out there. Any ideas on what to do? I can easily add an extra check to `policyId == 'default'`

Reply:
I'm running to existing usages. Let's see what was the intention of such usage.

Just to clarify, so the distinction is:
- `data-block-on-consent` leaves the consent handling to individual element (extension). Which by default is the same as `_if_accepted`. Element/extension can override this behavior.
- `data-block-on-consent=default` is considered as pub explicitly want to have control over consent handling. Which right now is also `_if_accepted`. Pub can override this by modifying amp-consent config policy `default`.

BTW, now I feel the naming `_if_accepted` is a bit confusing. `data-block-on-consent=til_accepted` is more accurate.

Reply:
renamed to `_till_accepted` and `_till_responded`

Reply:
To clarify the behavior: publisher always have the final words on consent behavior using `data-block-on-consent=_till_accepted/_till_responded/_auto_reject`. That will block component at `#buildCallback()`. 

On top of that, individual extension can customize their behaviors like what they does today.

`data-block-on-consent` and `data-block-on-consent='default'` are exactly the same thing. `data-block-on-consent` and `data-block-onconsent='_till_accepted` are the same thing if the publisher doesn't customize the `default` consent blocking behavior, and the component doesn't have its blocking behavior customized. 

I'll add all these to documentation once the feature is out.